### ISSUE TITLE
Boosted issue does not reflect on Agenda page

### ISSUE THREAD

**Issue by emilianodellacasa** [zerocracy/farm#1439]
Even though issue https://github.com/zold-io/zold/issues/98 has boosted 4x, in my Agenda page I still se the cost without boosting.

This is confusing: the boost should be visible everywhere, or I should be able to see in a glance from the Agenda page if the issue has been boosted.

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by 0crat** [comment]
@emilianodellacasa[/z](https://www.0crat.com/u/emilianodellacasa) this project will fix the problem faster if you donate a few dollars to it; just [click here](https://www.0crat.com/contrib/C3NDPUA8L) and pay via Stripe, it's very fast, convenient and appreciated; thanks a lot!

**Comment by g4s8** [comment]
@emilianodellacasa right, you have boost in https://github.com/zold-io/zold/issues/98#issuecomment-405658874 job, but it doesn't reflect your agenda.

**Comment by 0crat** [comment]
Job [#1439](https://github.com/zerocracy/farm/issues/1439) is now in scope, role is `DEV`
